### ISSUE TITLE
Installing doesn't seem to work

### ISSUE THREAD

**Issue by mstruebing** [editorconfig/editorconfig-core-go#9]
Hey, I wanted to make a small PR but I don't get it to work.

What I did:

```
go get -u github.com/editorconfig/editorconfig-core-go && \
go get go get -u gopkg.in/editorconfig/editorconfig-core-go.v1 && \
cd ~/go/github.com/editorconfig/editorconfig-core-go && \
make submodule && \
make installdeps
```

While executing `make installdeps` I get the following error:

```
$ make installdeps
go get -t ./...
# github.com/editorconfig/editorconfig-core-go/cmd/editorconfig
cmd/editorconfig/main.go:26:35: undefined: editorconfig.ConfigNameDefault
cmd/editorconfig/main.go:49:15: undefined: editorconfig.GetDefinitionForFilenameWithConfigname
make: *** [Makefile:12: installdeps] Error 2 
```

Is this already known? How can I fix this? Or is it something with my specific setup/usage?

Thanks in advance for your help!

**Comment by 10sr** [comment]
Sorry for inconvenience I've just released v1.3.0 .
It should be fixed after running `go get -u gopkg.in/editorconfig/editorconfig-core-go.v1` again.
Thanks!

**Comment by mstruebing** [comment]
Working now, thx!
